.gitignore

```diff
@@ -22,3 +22,6 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 replay_pid*
+
+# OneNote
+**/Notebook.onetoc2
```

Comment:
This isn't necessary and for code cleanliness, this should be removed

Reply:
However given the fact that it is not entirely relevant now and we can always focus on Clean Code post implementation. This can be ignored for now